shared/src/commonMain/kotlin/co/nimblehq/kmm/template/domain/repository/Repository.kt

```diff
@@ -0,0 +1,8 @@
+package co.nimblehq.kmm.template.domain.repository
```

Comment:
I think we should agree to keep the package name in plural 🙏 

```suggestion
package co.nimblehq.kmm.template.domain.repositories
```